README.md

```diff
@@ -112,25 +116,25 @@ of these methods accept planner-specific keyword arguments.
 ### Writing a Custom Planner
 
 Implementing a custom planner requires extending the `BasePlanner` class and
-decorating one or more methods with the `@PlanningMethod` decorator. Extending
-the `BasePlanner` class constructs the planning environment `self.env` and
-allows PrPy to identify your planner as a base planner class, as opposed to a
-meta-planner. The `@PlanningMethod` decorator handles environment cloning and
-allows meta-planners to query the list of planning methods that the planner
-supports (e.g. to generate docstrings).
+decorating one or more methods with the `@LockedPlanningMethod` or
+`@ClonedPlanningMethod` decorator. Extending the `BasePlanner` class constructs
+the planning environment `self.env` and allows PrPy to identify your planner
```

Comment:
What is the purpose of `self.env` in a planner that uses exclusively `@LockedPlanningMethod` decorators? You should note this here.